ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -0,0 +1,250 @@
+# English (Australia) translations for ckanext-qa.
```

Comment:
I really don't like using a pot translation file for changing the report names, it should really be a config options to change from ticks to star and vice versa..